release-notes/EPM/I-DOCSIS/EPM_7.0.0_I-DOCSIS.md

```diff
@@ -0,0 +1,125 @@
+
+#### Improved performance when navigating to EPM cards through Topology app [ID_38194]
+
+Performance has improved when you navigate to EPM cards through the Topology app. The Visual pages will now load faster and KPIs will no longer appear one at a time.
+
+#### Generic DOCSIS CM Collector now supports both comma and period as decimal separator [ID_38195]
+
+The Generic DOCSIS CM Collector connector can now parse numeric values regardless of whether a comma or period is used as decimal separator (invariant culture).
+
+#### Generic DOCSIS CM Collector: Improved calculation method for FEC values [ID_38196]
+
+The calculation method for Forward Error Correction (FEC) values in the Generic CM Collector was enhanced. Previously, FEC values were calculated using averages of channel instances, leading to potential discrepancies, particularly in scenarios where Uncorrectable Packet Ratio (UER) was high while the Corrected Packet Ratio (CER) showed as zero. Now, instead of relying on averages, the system aggregates all counters to accurately determine the ratio.
+
+#### Huawei 5688-5800 CCAP Platform connector now aligns CCAP name with element name [ID_38197]
+
+​The Huawei 5688-5800 CCAP Platform connector previously used a custom system name for its EPM CCAP name, which could cause mapping issues when retrieving element data. Now, the connector uses the element name as the CCAP name, resolving problems in Visio when looking up element-specific data.
+
+#### Skyline EPM Platform: DOCSIS 3.1 OFDM/OFDMA integration [ID_38198]
+
+DOCSIS 3.1 polling support is available, displaying all instances of OFDM/OFDMA channels from both the CCAP and CM perspectives. At both Node Segment and Service Group levels, you can now view all associated 3.1 channels and corresponding KPIs using the *OFDM channels* dashboard and *OFDMA channels* dashboard. You can also view all instances of CM 3.1 channels using the *CM OFDM channels* dashboard and *CM OFDMA channels* dashboard.
+
+#### Skyline EPM Platform: Maps now use relational filtering [ID_38199]
+
+Maps now employ relational filtering to identify all available map entities. This ensures a precise alignment with the data presented on the EPM card, enhancing data reliability.
+
+#### Improved OOS reporting: Define thresholds per channel [ID_38425]
+
+Improvements have been made to *Out of Spec (OOS)* reporting by allowing different thresholds for the Rx Power Status, SNR Status, and Post-FEC Status parameters for the US QAM and DS QAM channel modulations. Previously, one threshold was used globally across all channels. Now, thresholds vary depending on the modulation of the associated cable modem (CM) channel. Parameters that used to have fixed thresholds have been replaced with a new table containing the different modulations along with their corresponding thresholds.
+
+#### Separate procedures for adding a single CCAP/CM pair and creating elements in bulk [ID_38518]
```

Comment:
Can we move this RN to the "New features" section at the top?

Reply:
Hi @MariekeGO, yes, it is a new feature. Makes sense to move it to the 'New Features' section